feature-libs/product-configurator/rulebased/components/exit-button/configurator-exit-button.component.ts

```diff
@@ -46,7 +58,15 @@ export class ConfiguratorExitButtonComponent {
         take(1)
       )
       .subscribe((product) =>
-        this.routingService.go({ cxRoute: 'product', params: product })
+        this.routerData$.pipe(take(1)).subscribe((routerData) => {
```

Comment:
Could you please simplify this expression and only subscribe once on this.configRouterExtractorService
      .extractRouterData()?

Reply:
Done